releasenotes/notes/zipkin-datadog-host-ip-interpretation.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: telemetry
+issue:
+  - 27911
+releaseNotes:
+  - |
+    **Fixed** interpretation of $(HOST_IP) in Zipkin and Datadog tracer address.
```

Comment:
Is this a new feature or restoring an old method?

Reply:
nm, I was checking if this problem has existed in a *released* version, not just in main.
The attached bug does not state istio version, but we can assume it is 1.7.

Reply:
Yeah this issue should have existed since proxyConfig mount introduced. So I think it is 1.6.